examples/react-native-typesript/.flowconfig

```diff
@@ -0,0 +1,54 @@
+[ignore]
```

Comment:
Why do we need to configure Flow in a Typescript project?

Reply:
part of the react-native boilerplate. will delete